### PR TITLE
Tests are passing

### PR DESCRIPTION
**Pull request by Regggis**
Commented bonus, had some problems, normal tests done

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule-bonus
🏅 Granted 10 XP for @Regggis

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Regggis for on-time delivery

**Comment by ziobron** [comment]
✅ CI Passed - Task: schedule
🏅 Granted 12 XP for @Regggis

**Comment by ziobron** [comment]
⏱ 3 XP extra for @Regggis for on-time delivery

**Comment by Regggis** [comment]
10 points for bonus shouldn't be granted for now. Tests are commented, because of some problems with linking. I'm sorry for problem, i will try to do bonus task right now.

**Review by lukasz-kukulka — APPROVED** [review]

File: homework/schedule/schedule.hpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include <chrono>
+#include <functional>
+#include <string>
+#include <thread>
+
+template<typename f, typename T, typename... Args>
+void schedule(f ff, T time, Args... a) {
+    std::this_thread::sleep_for(time);
+    ff(a...);
```
Comment (nauka-programowania-MA) [inline homework/schedule/schedule.hpp:10]: use perfcet forwarding here. std::forward<Args>(args)...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**